assets/src/edit-story/components/canvas/layout.js

```diff
@@ -240,7 +240,10 @@ PageArea.propTypes = {
   children: PropTypes.node,
   showSafeZone: PropTypes.bool,
   showOverflow: PropTypes.bool,
-  fullbleedRef: PropTypes.object,
+  fullbleedRef: PropTypes.oneOfType([
```

Comment:
PropTypes.any may also be fine here

Reply:
How about object + func? https://github.com/google/web-stories-wp/pull/3935#discussion_r470786239

Reply:
I like that better, since it fits on one line and is more consistent with the rest of the codebase. Updated here: https://github.com/google/web-stories-wp/pull/4070

Adding any kind of type is kind of futile, since it's very difficult to pass anything other than a ref to a ref, since it's clearly labelled as a ref. I'm surprised PropTypes doesn't yet have its own type for refs though, perhaps we could add our own sometime for Story Editor.